packages/edit-post/src/components/header/fullscreen-mode-close/index.js

```diff
@@ -7,7 +7,11 @@ import { get } from 'lodash';
  * WordPress dependencies
```

Comment:
Apparently, this is the top left button in full screen mode, I can't get it to animate on hover? Am I missing something?

Reply:
Got it now, I was missing a site icon.